web-app/CKS.Web.Test/IntegrationTests/SearchTests.cs

```diff
@@ -11,19 +18,29 @@ public class SearchTests : TestBase
 		public SearchTests(ITestOutputHelper output) : base(output)
 		{ }
 
-		[Obsolete("Please replace this test with a real test when the search endpoint is setup")]
+
 		[Fact]
 		public async void ReturnsJson()
 		{
 			// Arrange
 			var client = _factory
-			   .CreateClient();
+				.WithFakeSearchProvider("FakeSearchResultsModel.json")
+				.CreateClient();
 
 			// Act
 			var response = await client.GetAsync("/api/search");
+			var responseBody = await response.Content.ReadAsStringAsync();
 
+			// Assert
 			response.StatusCode.ShouldBe(System.Net.HttpStatusCode.OK);
 			response.Content.Headers.ContentType.MediaType.ShouldBe("application/json");
+
+			var expected = JObject.Parse(File.ReadAllText(@"./IntegrationTests/Fakes/FakeSearchResultsModel.json"))
```

Comment:
Can we do `JToken.DeepEquals` here please? https://www.newtonsoft.com/json/help/html/DeepEquals.htm

Reply:
ignore that. Probably nicer as a string comparison anyway because of the diff we'd get if it failed